### PR TITLE
ignore anon user in analytics

### PR DESCRIPTION
**Pull request by norkans7**
Fixes https://sentry.io/organizations/nyaruka/issues/1532503323

### CODE REVIEW

**Comment by rowanseymour** [comment]
So we only call `analytics.track` when we have an authenticated user? Should we update that function to take a user object?

**Comment by norkans7** [comment]
I can try that too

File: temba/channels/tasks.py [hunk 1]
```diff
@@ -117,4 +117,4 @@ def track_org_channel_counts(now=None):
 
         for org in org_counts:
             if org.administrators.all():
-                track(org.administrators.all()[0].email, stat["key"], dict(count=org.count))
+                track(org.administrators.all()[0], stat["key"], dict(count=org.count))
```
Comment (rowanseymour) [inline temba/channels/tasks.py:120]: `.first()`

**Review by rowanseymour — APPROVED** [review]

File: temba/channels/tasks.py (hunk 1 above)
Comment (rowanseymour) [reply temba/channels/tasks.py]: sorry I just noticed that we're prefetching `administrators` so my suggestion creates new db hits - let's revert

Comment (norkans7) [reply temba/channels/tasks.py:120]: Yeah, there is a new DB hits

**Pull request merged**